browser/src/Editor/BufferManager.ts

```diff
@@ -295,6 +295,9 @@ export class Buffer implements IBuffer {
         }
 
         const getToken = (lineContents: string, character: number): Oni.IToken => {
+            if (!lineContents || !character) {
```

Comment:
Interesting, I wonder if this is related to an issue in my PR #1298... perhaps I'm hitting this too!